assets/js/blocks/cart-checkout/mini-cart-contents/inner-blocks/allowed-blocks.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * External dependencies
+ */
+import { getBlockTypes } from '@wordpress/blocks';
+
+const EXCLUDED_BLOCKS: readonly string[] = [
+	'woocommerce/mini-cart',
+	'woocommerce/single-product',
```

Comment:
I would also exclude `woocommerce/cart` and `woocommerce/checkout`.

I also noted there are three additional blocks which are available from the inserter but I expected them not to be: `woocommerce/checkout-order-note-block`, `woocommerce/checkout-terms-block` and `woocommerce/cart-accepted-payment-methods-block`. I'm a bit confused by them because they [specify a parent block](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/trunk/assets/js/blocks/cart-checkout/cart/inner-blocks/cart-accepted-payment-methods-block/block.json#L14), so I didn't expect them to be available in the Mini Cart context. :confused: Any clues?

Reply:
> I'm a bit confused by them because they specify a parent block, so I didn't expect them to be available in the Mini Cart context.

@Aljullu We only exclude child blocks of ones we set for `EXCLUDED_BLOCKS`. I added `woocommerce/cart-totals-block` and `woocommerce/checkout-fields-block` to the excluded blocks list. Those blocks and their child blocks are excluded now. Can you please take a look?